docs/API.AMQP.md

```diff
@@ -1227,6 +1228,7 @@ These statistics include:
 * the number of completed requests since the last frame
 * the number of failed requests since the last frame
 
+**Topic:** ``admin.getLastStat``
```

Comment:
should be ``getLastStats``, as it returns the last statistics frame (one frame containing multiple statistics)